src/aux-server/aux-web/aux-auth/site/AuthApp/AuthApp.vue

```diff
@@ -196,6 +196,18 @@
             </md-dialog-actions>
         </md-dialog>
 
+        <md-dialog :md-active.sync="showErrorDialog">
+            <md-dialog-title>Error</md-dialog-title>
+
+            <md-dialog-content>
+                <p>{{ errorMessage }}</p>
+            </md-dialog-content>
+
+            <md-dialog-actions>
+                <md-button class="md-primary" @click="closeErrorDialog">Close</md-button>
+            </md-dialog-actions>
+        </md-dialog>
+
```

Comment:
@TroyceGowdy Would it be possible to display the error inside the same dialog that the user submitted (instead of showing a new dialog with the error)? I think it would be nicer from a UX perspective if the user was able to see that there was an error and then be able to fix it immediately without having to open the "create record" dialog again.